src/languages/prolog.js

```diff
@@ -51,15 +51,13 @@ export default function(hljs) {
   };
 
   var CHAR_CODE = {
-
     className: 'string', // 0'a etc.
-    begin: /0\'(\\\'|.)/
+    begin: /0'(\\'|.)/
   };
 
   var SPACE_CODE = {
-
     className: 'string',
-    begin: /0\'\\s/ // 0'\s
+    begin: /0'\\s/ // 0'\s
```

Comment:
This is greek to me, but I think it's correct... anyone know Prolog?  The "\s" is literal?

Reply:
This makes sense to me now I think.  Code might be:

```prolog
% \s literal (escaped)
0'\s
% character "b"
0'b
% character "'" (escaped)
0'\'
```

Reply:
Yea, looks like `0'` is the way of getting the character code of a char. So `0'\s` returns `32`. And you are correct, `0''` is valid, it returns `39`

Screenshots and confirmation courtesy of @Adwitiya-Singh

![image](https://user-images.githubusercontent.com/1246453/97922952-1db5c280-1d12-11eb-8640-6840b4c904ac.png)

More info, https://www.swi-prolog.org/pldoc/man?section=charescapes

Reply:
> And you are correct, 0'' is valid, it returns 39

Valid or invalid?  Although I suppose the regex currently allows it either way, LOL... and I'm not even sure if that is bad or not.

Reply:
My bad, I misread your original comment 😅 `0''` is _valid_ as seen in the screenshot. But since the regex allows it, works for me